### PR TITLE
Fix to follow ethereum/wiki/wiki

### PR DESCRIPTION
**Pull request by zaq1tomo**

### CODE REVIEW

**Comment by zaq1tomo** [comment]
https://github.com/ethereum/wiki/wiki

**Comment by zaq1tomo** [comment]
@pirapira I have not written about Ether's investment. Where is the relevant part of your review? Also, why do you think about that?

File: Ethereum-Introduction.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+* Casper introductions:
+    * [Correct by construction Casper prototype](https://ethresear.ch/t/the-correct-by-construction-casper-paper-prototype-published-at-devcon-tear-it-apart/196)
+    * [Casper the Friendly Finality Gadget](https://ethresear.ch/t/latest-casper-basics-tear-it-apart/151/57)
+    * [Casper block proposer](https://ethresear.ch/t/initial-explorations-on-full-pos-proposal-mechanisms/925)
+* [The stateless client concept](https://ethresear.ch/t/the-stateless-client-concept/172)
+* [Ethereum 2 and alternative PoS implementations](https://ethresear.ch/t/ethereum-2-and-alternative-pos-implementations/190/7)
+
+# Concluding remarks
+
+Ether certainly seems like a good investment, and a good alternative to using fiat currencies, as well as an enabler for otherwise uneconomical business, due to lower transaction costs. It's more decentralized nature than central banks has advantages for trade from a local to global scale. With governance applications and systems on top Ethereum, it is even possible to do away with the hindering borders surmounted by nation-states. By doing away with these borders, society can be more open, inclusive and equitable.
```
Comment (pirapira) [inline Ethereum-Introduction.md]: This is the line recommending Ether as a good investment.  I disagree, and I oppose putting this here.

Comment (zaq1tomo) [reply Ethereum-Introduction.md]: Remove it.

**Comment by zaq1tomo** [comment]
Fix it.

**Comment by pirapira** [comment]
By the way, you can edit the wiki on your own.

**Pull request closed without merging**